### ISSUE TITLE
Pimcore 6.6.7: Invalid Cached Object Signature 

### ISSUE THREAD

**Issue by solverat** [pimcore/pimcore#6660]
Also reproducable on demo:

```php

$obj = \Pimcore\Model\DataObject\News::getById(772);

// clear data cache
dump($obj instanceof \Pimcore\Model\DataObject); // true

// reload
dump($obj instanceof \Pimcore\Model\DataObject); // false

**Comment by brusch** [comment]
Do you have more infos on how exactly to reproduce it? Thanks a lot!

**Comment by solverat** [comment]
@brusch sure. 

Just add this to your demo controller:

```php
// AppBundle\Controller\ContentController
public function defaultAction()
{
     $obj = \Pimcore\Model\DataObject\News::getById(771);
     var_dump($obj instanceof \Pimcore\Model\DataObject); // false
     exit;
}
```

save it, then open https://demo.pimcore.fun/en/Events 

I first opened `en/Magazine` => which returns `true`. Mysterious! Maybe a runtime issue...

**Comment by solverat** [comment]
For the record, locally I've disabled `full_page_cache´ to be sure it's not a FE cache issue.

**Comment by dvesh3** [comment]
resolved by #6664
